shor/quantum.py

```diff
@@ -53,6 +53,19 @@ def measure_bits(self):
 
         return measure_bits
 
+    def to_registers(self):
+        registers = []
```

Comment:
We probably want the output of this function to be:

qbit_registers, cbit_registers

Where each one is a list.

Reply:
Initialize like so:
qbit_registers, cbit_registers = [], []